### PR TITLE
chore(build): Add data-testid attributes to npm and production builds

### PR DESCRIPTION
**Pull request by raghulsid**
This is for retaining test tags in the repo's. 
Have removed the line of code that was striping the data-testid from production and npm builds. 
By retaining the data-testId we could have a better strategy for element locator which would help us to avoid test caused by selector moves or subtly changes on screen.

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -47,12 +47,6 @@ module.exports = {
         development: {
             plugins: ['flow-react-proptypes'],
         },
-        npm: {
-            plugins: [['react-remove-properties', { properties: ['data-testid'] }]],
-        },
-        production: {
-            plugins: [['react-remove-properties', { properties: ['data-resin-target', 'data-testid'] }]],
```
Comment (jstoffan) [inline babel.config.js:54]: Should `data-resin-target` attributes still be removed? That change seems unrelated to `data-testid`. If not, the `react-remove-properties` dependency should also be removed.

**Comment by alexkrolick** [comment]
@raghulsid you'll need to update your commit message to make the commit bot happy and help us automate the release: https://github.com/box/box-ui-elements/blob/master/CONTRIBUTING.md#commit-types

**Comment by jstoffan** [comment]
@raghulsid, please update the commit messages of each commit (squash and force push first to make this easier) and the PR description to something like `chore(build): Add data-testid attributes to npm and production builds`.

**Comment by raghulsid** [comment]
> @raghulsid, please update the commit messages of each commit (squash and force push first to make this easier) and the PR description to something like `chore(build): Add data-testid attributes to npm and production builds`.

**Comment by jstoffan** [comment]
@raghulsid, please seek @box/ui-elements approval before adding the ready-to-merge label.
